ghost/email-service/lib/email-event-storage.js

```diff
@@ -61,7 +66,11 @@ class EmailEventStorage {
     }
 
     async handleComplained(event) {
-        return this.unsubscribeFromNewsletters(event);
+        await this.#EmailSpamComplaintEvent.add({
```

Comment:
Awaiting discussion about duplicate events, this might need protection to prevent adding the same event multiple times in the db

Reply:
Going to add protection for now - good shout!